Documentation/ContentElements/Content/Index.rst

```diff
@@ -65,3 +65,9 @@ Highlighted areas will then show you where you can drop it.
 
 You can move a content element from one page to another in the :guilabel:`Web
 > List` module using cut and paste. Refer to the :ref:`clipboard chapter <clipboard>` for information about using the clipboard in the :guilabel:`List` module.
+
+.. tip::
+
+   If you are more comfortable using shortcuts, there are different kinds of
+   :ref:`keyboard commands <t3editors:keyboard_commands>` you can use in
+   the Backend Editor.
```

Comment:
Maybe also use "backend forms" here, too?

Reply:
thanks for the hint. I will change it in this parts as well